pom.xml

```diff
@@ -203,51 +202,51 @@
     <jakarta.servlet-api.version>6.0.0</jakarta.servlet-api.version>
     <jakarta.validation.version>3.0.2</jakarta.validation.version>
     <jakarta.websocket-api.version>2.1.1</jakarta.websocket-api.version>
-    <jakarta.xml.bind-api.version>4.0.0</jakarta.xml.bind-api.version>
+    <jakarta.xml.bind-api.version>4.0.1</jakarta.xml.bind-api.version>
     <jakarta.xml.soap-api.version>3.0.0</jakarta.xml.soap-api.version>
     <javapoet.version>1.11.1</javapoet.version>
-    <jaxb.version>4.0.2</jaxb.version>
-    <jetty.version>11.0.15</jetty.version>
+    <jaxb.version>4.0.3</jaxb.version>
+    <jetty.version>12.0.1</jetty.version>
```

Comment:
looks like this is not present in central: https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-servlet/11.0.16. maybe I've misread the log. let me check.